crates/turborepo-lib/src/run/summary/global_hash.rs

```diff
@@ -65,6 +66,7 @@ impl<'a> TryFrom<GlobalHashableInputs<'a>> for GlobalHashSummary<'a> {
             files: global_file_hash_map.into_iter().collect(),
             // This can be empty in single package mode
             hash_of_external_dependencies: root_external_dependencies_hash.unwrap_or_default(),
+            hash_of_internal_dependencies: root_internal_dependencies_hash.unwrap_or_default(),
```

Comment:
As Github is annotating, I don't see `root_internal_dependencies_hash` in scope here